src/Parser.js

```diff
@@ -20,6 +20,7 @@ class Parser {
     this.registerExtension('vue', './assets/VueAsset');
     this.registerExtension('json', './assets/JSONAsset');
     this.registerExtension('json5', './assets/JSONAsset');
+    this.registerExtension('json-ld', './assets/JSONLDAsset');
```

Comment:
let's make the extension `.jsonld` as specified here: https://json-ld.org/spec/latest/json-ld/#iana-considerations

Reply:
👍